mail/isync/Portfile

```diff
@@ -60,8 +59,11 @@ post-destroot {
     xinstall -m 644 -W ${worksrcpath} mbsync.plist ${destroot}${examples_dir}/${startupitem.plist}
 }
 
-configure.cppflags-append   -I${prefix}/include/db53
-configure.ldflags-append    -L${prefix}/lib/db53
+variant bdb description {Enable support for the alternative UID storage scheme} {
```

Comment:
The variant should be enabled by default (with `default_variants`).

Reply:
done